docs/source/gettingstarted.rst

```diff
@@ -188,6 +189,21 @@ They are:
 Any combination of these arguments can be given. Omitting them all is fine too.
 Their behavior is documented in detail below.
 
+``max_y``
+^^^^^^^^^
+
+Specifies the maximum active twitch force to be set in the output xlsx file.
```

Comment:
This line is bit confusing. It's clear from the following lines what this argument does so I think we can just remove this line